docker-compose.yml

```diff
@@ -91,31 +91,6 @@ services:
         awslogs-group: ${LOG_GROUP_NAME}
         awslogs-stream: "nginx_http"
 
-
-  nginx_dockstore:
-    build:
-      context: .
-      dockerfile: config/Dockerfile_ui2
-      args:
-        CI: ${CI}
-    restart: always
-    depends_on:
-      - webservice
-    ports:
-      - "8443:8080"
```

Comment:
I think this might be too broad of a deletion.
 
My understanding of the way we have cloudfront setup, cloud front has two origins, the S3 bucket and the this set of containers running on the VM at HTTPS 443 (and HTTP 80 if we didn't have a permanent redirect setup). 

Nginx is used as a reverse proxy serving up content on port 443, without the webservice being directly exposed (line 58 above) Killing nginx totally on port 443 would give CloudFront nothing to talk to. 

We could instead, expose the webservice directly but I think that may change the behavior of the webservice setup in nginx like https://github.com/dockstore/compose_setup/blob/develop/templates/default.nginx_https.shared.conf.template#L31 or https://github.com/dockstore/compose_setup/blob/develop/templates/default.nginx_https.conf.template#L2 or https://github.com/dockstore/compose_setup/blob/develop/templates/default.nginx_https.conf.template#L27

So I think a lower-impact change would be to strip out just the building of the UI from `Dockerfile_ui2` and the serving out of the UI at https://github.com/dockstore/compose_setup/blob/develop/templates/default.nginx_https.conf.template#L69 but this requires testing or just trying it out on staging. 

Side note: I think our migration will break compatibility with 8443, which makes it a good thing Terra has already moved on to 443.

Reply:
Ah, you're right. I was hasty and assumed this was for building the UI only.

And yes, it does look like we will [lose port 8443](https://serverfault.com/questions/964096/how-to-have-cloudfront-distribution-listen-on-custom-port-for-https).

I'm worried about Dockstore CLI, if users have that written out in their ~/.dockstore/config files.